manim/utils/cfgwriter.py

```diff
@@ -0,0 +1,81 @@
+from os import path
+import configparser
+
+from .config_utils import successfully_read_files
+
+from rich.console import Console
+from rich.progress import track
+from rich.color import Color
+
+
+def check_valid_colour(color):
+    """Checks whether the entered color is a valid color according to rich
+    Parameters
+    ----------
+    color : :class:`str`
+        The color to check whether it is valid.
+    Returns
+    -------
+    Boolean
+        Returns whether it is valid color or not.
+    """
+    try:
+        Color.parse(color) is None
+        return True
+    except:
+        return False
+
+
+def main():
+    console = Console()
+    config = configparser.ConfigParser()
+    config.read(
+        successfully_read_files
+    )
+    default = {
+        "logging.keyword": "bold yellow",
+        "logging.level.notset": "dim",
+        "logging.level.debug": "green",
+        "logging.level.info": "blue",
+        "logging.level.warning": "red",
+        "logging.level.error": "red bold",
+        "logging.level.critical": "red bold reverse",
+        "log.level": "",
+        "log.time": "cyan dim",
+        "log.message": "",
+        "log.path": "dim",
+    }
```

Comment:
There is no reason to define this again since these all are (or should all be) in the `default.cfg` file, under the logger section.